### ISSUE TITLE
Window scaling in Win10

### ISSUE THREAD

**Issue by teedoubleuu** [godotengine/godot#9275]
**Operating system or device - Godot version:**
Windows 10
Godot 2.1.3

**Issue description:**
Scaling option in Windows is set to 125%. Game window is upscaled messing up the resolution - if the game is setup to run at native resolution of the screen then part of the window is outside of the screen.

**Steps to reproduce:**
Set Windows upscaling to something greater than 100% and run a game in fullscreen or borderless window with the size of the screen.

**Comment by fire** [comment]
Also in godot 3.0.

https://i.imgur.com/SeB4VQW.png

The screenshot shows that the position of the cursor is not where the editor thinks it is.

**Comment by vnen** [comment]
Related to #6601.

**Comment by myfreeweb** [comment]
Trying out 3.0-alpha1.

The executable is not marked as DPI aware!! Have to override in properties:

<img width="270" alt="2017-07-27 15_53_43-godot_v3 0-alpha1_win64 exe properties" src="https://user-images.githubusercontent.com/208340/28671008-d4498f86-72e3-11e7-9e41-d92cc9462faf.png">

The correct scale is not selected by default, and changing it in the menu does not instantly apply it, have to restart the editor.

<img width="1341" alt="2017-07-27 15_54_32-godot engine - platformer 3d - stage scn" src="https://user-images.githubusercontent.com/208340/28671054-f1ab1eaa-72e3-11e7-928d-350bb79c925d.png">

And uhhh the font rendering in MidDPI is weirdly stretched.

In summary, Godot:

- should be marked as DPI aware
- should automatically select the "Hidpi Mode" that matches the scale that Windows provides
- should render the text better
- (ideally) should apply the "Hidpi Mode" instantly

**Comment by Calinou** [comment]
Godot only really supports 100% and 200% scaling, 75% and 150% are going to look ugly because the icons aren't generated for those sizes, and we don't have 125% and 175% options yet. (If we followed what Windows 10 provides, then 225%, 250% and 300% should be provided as well.)


Also, marking the executable as DPI-aware has been known to cause issues with Intel HD 3000 graphics, but since these are not going to be supported in Godot 3.0, we might as well mark Godot as DPI-aware.

**Comment by myfreeweb** [comment]
Yeah icons look slightly distorted but whatever, who cares about them. Just fix the font rendering at 150%, text is much more important than icons

**Comment by vnen** [comment]
Fixed by eb0f9651b3bb12b79db4ee4f8862a3be16a11ea5
